.github/workflows/ci.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     env:
       PYTHON_VERSION: ${{ matrix.python-version }}
       PARALLEL: "true"
-      COVERAGE: "false"
+      COVERAGE: "true"
```

Comment:
Does coverage work when running tests in parallel?

Reply:
Whoops it doesn't out of the box. Should be fixed now.